zeebe/logstreams/src/test/java/io/camunda/zeebe/logstreams/util/SyncLogStreamBuilder.java

```diff
@@ -66,6 +67,12 @@ public SyncLogStreamBuilder withLogName(final String logName) {
     return this;
   }
 
+  @Override
+  public LogStreamBuilder withMeterRegistry(final MeterRegistry meterRegistry) {
+    delegate.withMeterRegistry(meterRegistry);
+    return null;
```

Comment:
:question: why null? shouldn't be `this` ?

Reply:
yes :see_no_evil: